docs/es/mkdocs.yml

```
@@ -37,7 +37,7 @@ plugins:
     data: data
 nav:
 - FastAPI: index.md
-- Languages:
+- Idiomas:
```

```suggestion
- Languages:
```

This one has to be kept in English.